batchlib/analysis/cell_level_analysis.py

```diff
@@ -604,6 +629,112 @@ def validate_outputs(self, output_files, folder, status, ignore_failed_outputs):
             return have_table
 
 
+def add_site_name_to_image_table(column_names, table):
+    assert column_names[0] == 'image_name'
+    column_names = list(column_names)
+    column_names.insert(1, 'site_name')
+    table = np.array(table).tolist()
+    for row in table:
+        row.insert(1, image_name_to_site_name(row[0]))
+    return column_names, np.array(table)
+
+
+class ExtractBackground(CellLevelAnalysisWithTableBase):
+    def __init__(self,
+                 cell_seg_key, serum_key, marker_key,  # TODO: get rid of this in CellLevelAnalysisWithTableBase
+                 actual_channels_to_use,
+                 image_outlier_table='images/outliers',
+                 identifier=None,
+                 **super_kwargs):
+
+        self.channel_keys = actual_channels_to_use
+        self.image_outlier_table = image_outlier_table
+
+        group_name = 'backgrounds' if identifier is None else f'backgrounds_{identifier}'
+        self.image_table_key = 'images/' + group_name
+        self.well_table_key = 'wells/' + group_name
+        self.plate_table_key = 'plate/' + group_name
+
+        super(ExtractBackground, self).__init__(
```

Comment:
I prefer new style super a lot, which is just `super().__Init__(...)`. Could you change it?